### PR TITLE
Dlc 762 sites with local search type

### PR DESCRIPTION
**Pull request by barmintor**

### CODE REVIEW

File: app/assets/javascripts/sites/edit.js [hunk 1]
```diff
@@ -8,15 +8,15 @@ function addSiteImageUriFieldSet(addButtonFieldset) {
...
+	newInput['id'] = 'site_image_uris[' + newIndex + ']';
 	newFieldset.appendChild(newInput);
 	// <input type="button" class="btn btn-danger" value="Remove" onclick="this.parentElement.remove();"/>
 	if (newIndex > 0) {
 		const newButton = document.createElement("input");
 		newButton['type'] = 'button';
 		newButton['className'] = 'btn btn-danger';
 		newButton['value'] = 'Remove';
-		newButton['onclick'] = () => this.parentElement.remove();
+		newButton['onclick'] = function() { this.parentElement.remove() };
```
Comment (elohanlon) [inline app/assets/javascripts/sites/edit.js:19]: Yeah, unfortunately without babel we'll need to stick with old fashioned functions for IE 11 compatibility.

Comment (barmintor) [reply app/assets/javascripts/sites/edit.js:19]: it was sad times when I deployed the earlier version :-(

File: spec/routing/sites/pages_controller_spec.rb [hunk 2]
```diff
@@ -34,6 +34,11 @@
       expect(:get => "/foo/pages").to route_to(controller: "sites/pages", action:"index", site_slug: 'foo')
       expect(:get => "/restricted/foo/pages").to route_to(controller: "restricted/sites/pages", action:"index", site_slug: 'foo')
     end
+    it "does not route asset paths to pages" do
+      { 'images' => 'jpg', 'javascripts' => 'js', 'stylesheets' => 'css' }.each do |asset_type, file_ext|
```
Comment (elohanlon) [inline spec/routing/sites/pages_controller_spec.rb:38]: Very good test to have 👍

File: spec/routing/sites/search_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe Sites::SearchController, :type => :routing do
```
Comment (elohanlon) [inline spec/routing/sites/search_controller_spec.rb:3]: More great tests here 👍

**Review by elohanlon — APPROVED** [review]
Other than my two comments in search_controller.rb, this looks good!

**Pull request merged**